lumicks/pylake/calibration.py

```diff
@@ -1,3 +1,218 @@
+import re
+from collections import UserDict
+
+
+def _read_from_bl_dict(bl_dict):
+    mapping = {
```

Comment:
simple docstring might be nice, just so you don't have to think about all the different `dict`s being passed around

Reply:
👍